TwilioVerifySDK/TwilioVerify/Sources/Models/ChallengeListPayload.swift

```diff
@@ -37,12 +39,20 @@ public struct ChallengeListPayload {
     - pageSize: Number of Challenges to be returned by the service
     - status: Status to filter the Challenges, if nothing is sent, Challenges of all status will be
               returned
+    - order: Sort challenges in order by creation date of the challenge
     - pageToken: Token used to retrieve the next page in the pagination arrangement
   */
-  public init(factorSid: String, pageSize: Int, status: ChallengeStatus? = nil, pageToken: String? = nil) {
+  public init(factorSid: String, pageSize: Int, status: ChallengeStatus? = nil, order: ChallengeListOrder = .asc, pageToken: String? = nil) {
     self.factorSid = factorSid
     self.pageSize = pageSize
+    self.order = order
     self.status = status
     self.pageToken = pageToken
   }
 }
+
+///Describes the order to fetch a **ChallengeList**
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/9cbd540c6135af3d1641ad17aef5ccc5e04d146c/TwilioVerifySDK/TwilioVerify/Sources/Models/ChallengeListPayload.swift#L54"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>ChallengeListPayload.swift:54</code></td>
    </tr>
  </tbody>
</table>